src/poll.rs

```diff
@@ -53,7 +119,32 @@ pub fn poll(fds: &mut [PollFd], timeout: libc::c_int) -> Result<libc::c_int> {
     Errno::result(res)
 }
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
+/// `ppoll()` allows an application to safely wait until either a file
```

Comment:
Ok, I wanted to focus on the difference between `poll` and `ppoll`. Should I juste put the first line of man, and redirect to the web page for more info ?